Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -157,14 +157,46 @@ public virtual void RenderCustomInspector()
             {
                 position = EditorGUILayout.GetControlRect();
                 DrawDropDownProperty(position, actionId, actionOptions, new GUIContent("Input Actions", "The input action filter"));
-
             }
 
+            EditorGUI.indentLevel = indentOnSectionStart + 1;												 
             SerializedProperty isGlobal = serializedObject.FindProperty("IsGlobal");
             EditorGUILayout.PropertyField(isGlobal, new GUIContent("Is Global", "Like a modal, does not require focus"));
-
+            EditorGUI.indentLevel = indentOnSectionStart;
             SerializedProperty voiceCommands = serializedObject.FindProperty("VoiceCommand");
-            EditorGUILayout.PropertyField(voiceCommands, new GUIContent("Voice Command", "A voice command to trigger the click event"));
+
+            // check speech commands profile for a list of commands
+            if (speechKeywords == null)
+            {
+                GUI.enabled = false;
+                EditorGUILayout.Popup("Speech Command", 0, new string[] { "Missing Speech Commands" });
+                GUI.enabled = true;
+            }
+            else
+            {
+                //look for items in the sppech commands list that match the voiceCommands string
+                // this string should be empty if we are not listening to speech commands
+                // will return zero if empty, to match the inserted off value.
+                int currentIndex = SpeechKeywordLookup(voiceCommands.stringValue, speechKeywords);
+
+                position = EditorGUILayout.GetControlRect();
+                GUIContent label = new GUIContent("Speech Command", "speech keyword to trigger Interactable");
+                EditorGUI.BeginProperty(position, label, voiceCommands);
+                {
+                    currentIndex = EditorGUI.Popup(position, label.text, currentIndex, speechKeywords);
```

Comment:
>These values only work if they are set before runtime or OnEnable is called. The Interactable does not currently support runtime registering and unregistering.

We should put the dropdown in a disabled block during play mode.

Reply:
I disabled some of the settings during play mode, those that should only be edited in edit mode.